### ISSUE TITLE
1.8.8: java.lang.NullPointerException: Cannot invoke "java.lang.reflect.Method.setAccessible(boolean)" because "buildCommandMethod" is null

### ISSUE THREAD

**Issue by kangarko** [EssentialsX/Essentials#5994]
### `/ess dump all` output

https://essentialsx.net/dump.html?id=c2cd37315b7044129a54e645624c5d7f

### Error log (if applicable)

https://paste.gg/p/anonymous/cd45fef5773549a68521ff93426a0b79

### Bug description

The above error log appears on 1.8.8 Paper and Panda in server startup log. I understand the server version is unsupported. I would appreciate at least converting the stack trace into a single line warning for convenience since the version is still listed on your website as a usable one. 

The plugin appears to be working normally except for this error.

Thanks!
Matej

**Comment by JRoy** [comment]
fixed this in #5087

**Comment by kangarko** [comment]
Thank you so much!

**Comment by kangarko** [comment]
I see it's been opened for 2 years, did the fix get pulled or is the fix only in the pull request?

**Comment by JRoy** [comment]
> I see it's been opened for 2 years, did the fix get pulled or is the fix only in the pull request?

yes it's an old pr but it's being planned to be merged very soon on our internal roadmap. Gonna leave this open until then

**Comment by kangarko** [comment]
Understood. Thanks for the explanation and happy holidays!
